website/src/docs/hotchocolate/integrations/spatial-data.md

```diff
@@ -478,7 +478,7 @@ input GeometryIntersectsOperationFilterInput {
         intersects: {
           geometry: {
             type: LineString,
-            coordinates: [[1, 1], ....]
+            coordinates: [[[1, 1], [2, 2]....]]
```

Comment:
hmm, i dont think this is the correct format for `LineString` 
https://docs.mapbox.com/android/java/api/libjava-geojson/5.8.0/com/mapbox/geojson/LineString.html

Reply:
You are 100 % correct. My apologies. I work with Polygon intersections that use the extra bracket, of course, and confused the types. Thanks @PascalSenn!